src/nupic/algorithms/Segment.cpp

```diff
@@ -351,6 +351,8 @@ void Segment::freeNSynapses(UInt numToFree,
                             std::vector<UInt>& removed, UInt verbosity,
                             UInt nCellsPerCol, Real permMax)
 {
+  // ZZZ inactiveSynapseIndices, activeSynapseIndices, and nCellsPerCol are not
```

Comment:
Do you want to create an issue for doing this or is it a simple change to make here? Might require changes to SWIG bindings and nupic.